src/org/labkey/test/Locators.java

```diff
@@ -21,7 +21,6 @@
     public static final Locator.IdLocator folderMenu = Locator.id("folderBar");
     public static final Locator.XPathLocator labkeyError = Locator.byClass("labkey-error");
     public static final Locator.XPathLocator labkeyMessage = Locator.byClass("labkey-message");
-    public static final Locator.XPathLocator alertWarning = Locator.byClass("alert").withClass("alert-warning");
```

Comment:
Just curious why this is being removed.

Reply:
It was unused and is probably redundant with `BootstrapLocators.warningBanner`